ci/verify_bundle.py

```diff
@@ -46,73 +88,84 @@ def verify_metadata_format(bundle_path: str):
     )
 
 
-def verify_data_shape(inference_file: str, bundle_path: str):
+def verify_data_shape(bundle_path: str, net_id: str, config_file: str):
     """
     This function is used to verify the data shape of network.
 
     """
     verify_net_in_out(
-        net_id="network_def",
+        net_id=net_id,
         meta_file=os.path.join(bundle_path, "configs/metadata.json"),
-        config_file=os.path.join(bundle_path, f"configs/{inference_file}"),
+        config_file=os.path.join(bundle_path, config_file),
         bundle_root=bundle_path,
     )
 
 
-def verify_export_torchscript(inference_file: str, bundle_path: str):
+def verify_export_torchscript(bundle_path: str, net_id: str, config_file: str):
     """
     This function is used to verify if the checkpoint is able to torchscript.
 
     """
     ckpt_export(
-        net_id="network_def",
+        net_id=net_id,
         filepath=os.path.join(bundle_path, "models/verify_model.ts"),
         ckpt_file=os.path.join(bundle_path, "models/model.pt"),
         meta_file=os.path.join(bundle_path, "configs/metadata.json"),
-        config_file=os.path.join(bundle_path, f"configs/{inference_file}"),
+        config_file=os.path.join(bundle_path, config_file),
         bundle_root=bundle_path,
     )
 
 
+def get_net_id_config_name(bundle_name: str):
+    """
+    Return values of arguments net_id and config_file.
+    """
+    net_id = "network_def"
+    config_file = "configs/inference.json"
+    if bundle_name not in custom_net_config_dict.keys():
+        return net_id, config_file
+
+    name_dict = custom_net_config_dict[bundle_name]
+    if "net_id" in name_dict.keys():
+        net_id = name_dict["net_id"]
+    if "config_file" in name_dict.keys():
+        config_file = name_dict["config_file"]
+
+    return net_id, config_file
+
+
 def main(changed_dirs):
     """
     main function to process all changed files. It will do the following steps:
 
     1. according to changed directories, get changed bundles.
-    2. verify each bundle on download large files, metadata format, data shape of network
-        and torchscript export (if supports).
+    2. verify each bundle.
 
     """
     bundle_list = get_changed_bundle_list(changed_dirs)
     models_path = "models"
 
     if len(bundle_list) > 0:
         for bundle in bundle_list:
-            bundle_path = os.path.join(models_path, bundle)
-
-            # verify download
-            verify_download_large_files(bundle_path)
+            # verify bundle directory
+            verify_bundle_directory(models_path, bundle)
```

Comment:
I would suggest to print some success log after every test case, it can be helpful when we check the error log in CI.

Thanks.